src/gawronski-bartosz/Fsd.Bartek.Cs/Fsd.Bartek.Cs.Ex2/Domain/Fabric.cs

```diff
@@ -0,0 +1,37 @@
+﻿using Fsd.Bartek.Cs2.Ex2.Domain.Animals;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Fsd.Bartek.Cs2.Ex2.Domain.UI;
+
+namespace Fsd.Bartek.Cs2.Ex2.Domain
+{
+    public class Fabric
+    {
+        public List<Animal> Animals = new List<Animal>();
```

Comment:
jeżeli to jest pole to prywatne i _animals. Jeżeli propercja to dodaj {get; private set;}